static/app/views/dashboards/widgets/lineChartWidget/lineChartWidget.stories.tsx

```diff
@@ -88,3 +110,28 @@ export default storyBook(LineChartWidget, story => {
 const MediumWidget = styled('div')`
   width: 420px;
 `;
+
+function filterTimeSeriesToSelection(
+  timeSeries: TimeseriesData,
+  start: DateString | null,
+  end: DateString | null
+): TimeseriesData {
+  return {
+    ...timeSeries,
+    data: timeSeries.data.filter(datum => {
+      if (start) {
+        if (moment(datum.timestamp).isBefore(moment.utc(start))) {
+          return false;
+        }
+      }
+
+      if (end) {
+        if (moment(datum.timestamp).isAfter(moment.utc(end))) {
+          return false;
+        }
+      }
```

Comment:
Is datum.timestamp already utc here?

Unless you expect other conditions, you could just shove it all into a single check.
```suggestion
        if (start && moment(datum.timestamp).isBefore(moment.utc(start))) {
          return false;
        }

        if (end && moment(datum.timestamp).isAfter(moment.utc(end))) {
          return false;
        }
```

Reply:
Ah, no, `datum.timestamp` is not in UTC, it's a timestamp with a timezone like `"2024-10-24T15:00:00-04:00"`